ocean_lib/test/test_example_config.py

```diff
@@ -0,0 +1,54 @@
+#
```

Comment:
I'd like to see unit tests for rinkeby, ropsten, and mainnet as well. After you integrate Calina's changes to the `get_web3_connection_provider` from https://github.com/oceanprotocol/ocean.py/pull/421, this should be possible.

Reply:
oh I now realise that part is not implemented. @mariacarmina even though you started from main and we decided to integrate the changes manually, step by step, instead of cherry picking and fixing conflicts, we still need to integrate the changes to get_web3_connection_provider.

Reply:
I will use the public RPCs for the unit tests.

Reply:
It only worked for Polygon. Rinkeby, ropsten and mainnet are infura nodes which provide by default an infura URL that contains the secret key (saw on MetaMask).